CHANGELOG.md

```diff
@@ -5,64 +5,51 @@ For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/suppor
-    room.disconnect();
-  });
-  ```
+  APIs are now generally available.
+- twilio-video.js now supports versions of Safari that enable Unified Plan as
+  the default SDP format. As of now, Unified Plan is enabled by default in the latest
+  Safari Technology Preview. (JSDK-2306)
+- Previously, Participants would be disconnected from the Room during network disruptions or handoffs. Now, you can now enable a new opt-in feature that enables Participants to remain connected to the Room.
+
+  To try this new feature in your application **you must perform the following steps**:
+
+  1. Set the Time-To-Live(TTL) of your [AccessToken](https://www.twilio.com/docs/video/tutorials/user-identity-access-tokens) to the maximum allowed session duration, currently 14400 seconds (4 hours). This ensures that when a network loss occurs the client will be able to re-authenticate with the signaling server. Failure to set a sufficiently long TTL may result in an [AccessTokenExpiredError](https://www.twilio.com/docs/api/errors/20104) when the client attempts to reconnect.
+  2. Ensure that the [AccessToken]((https://www.twilio.com/docs/video/tutorials/user-identity-access-tokens)) does not contain a configuration profile sid. Configuration profiles were deprecated
+    when we [announced](https://www.twilio.com/blog/2017/04/programmable-video-peer-to-peer-rooms-ga.html#room-based-access-control) the general availability of twilio-video.js@1.0.0 and are not supported when using this feature.
+  3. Ensure that the `identity` field provided in the [AccessToken](https://www.twilio.com/docs/video/tutorials/user-identity-access-tokens#generate-helper-lib) is a string. Using a non-string value will result in an [AccessTokenInvalidError](https://www.twilio.com/docs/api/errors/20101).
+  4. Enable the feature using the temporary flag `_useTwilioConnection` as follows:
+
+	  ```js
+	  const { connect } = require('twilio-video');
+	  const room = await connect(token, {
+	    _useTwilioConnection: true
+	  });
+	  ```
+  5. The reconnecting event will raise a [Signaling connection disconnected error](https://www.twilio.com/docs/api/errors/53001) when a signaling connection network disruption occurs. Previously, the reconnecting event only raised a [Media connection failed or ceased error](https://www.twilio.com/docs/api/errors/53405). You can differentiate between errors in the handler as follows:
+
+	  ```js
+	  room.on('reconnecting', error => {
+	    if (error.code === 53001) {
+	      console.log('Reconnecting your signaling connection!', error.message);
+	    } else if (error.code === 53405) {
+	      console.log('Reconnecting your media connection!', error.message);
+	    }
+	  });
+	  ```
+  6. To ensure that the total time to detect a participant leaving the room is minimized it is recommended that you disconnect from the room whenever the tab/browser is closed or when a page navigation event occurs. This can be acommplished as follows:
+
+	  ```js
```

Comment:
@idelgado Nitpick: Can we indent the code sample block to align with the bullet point?